src/utils/axiosSetup.ts

```diff
@@ -40,10 +39,11 @@ export const createResponseRejectedInterceptor = (interceptorParams: Interceptor
       let res: AxiosResponse<TokenResponse>;
       try {
         res = await interceptorParams.state.refreshingToken;
-      } catch (error: unknown) {
+      } catch (e) {
+        const error = e as AxiosError;
         // Check if refresh token expired, call tokenRefreshFailed() and then re-throw error
-        const status = (error as { raw?: { response?: { status?: number } } }).raw?.response?.status;
-        if (status === 497) {
+        const status = error.code;
+        if ((status ?? '').startsWith('497')) {
```

Comment:
if(stats?.startsWith('497') is not sufficient? Or what is status type?

Reply:
Why even startsWith?

Reply:
It's `string | undefined`, I'm using starsWith just to be sure: 
![image](https://user-images.githubusercontent.com/23738705/117982283-2678f800-b336-11eb-89ad-5e4b4a455a64.png)

Reply:
I believe the status itself is always 497 (integer) and this is added by postman (thats why its unknown, or OK for 200). So i would keep it ===.

Taken from axios docs:
`{
  // `status` is the HTTP status code from the server response
  status: 200,

  // `statusText` is the HTTP status message from the server response
  statusText: 'OK',
}
`